docs/user/dashboard/lens.asciidoc

```diff
@@ -371,7 +371,7 @@ The following component menus are available:
 
 When creating or editing a visualization, you can customize the way the legend gets displayed, and the data it displays. To do that, look for the image:images/legend-icon.svg[Legend icon] icon.
 
-image::images/legend-popover.png[Menu with options to customize the legend of a visualization, width=40%]
+image::images/legend-popover-8.16.0.png[Menu with options to customize the legend of a visualization, width=50%]
```

Comment:
```suggestion
image::images/legend-popover-8.16.0.png[Menu with options to customize the legend of a visualization, width=50%]
```
I remember updating this one specifically for 8.16 with options to customize the legend with additional statistics, which I don't see in your new screenshot. We might want to keep the previous one here.

Reply:
Good catch! I updated the screenshot to show the new icons in the workspace, but I've now added a new screenshot to show the icons as well as the statistics.